examples/platforms/efr32/sleepy-demo/README.md

```diff
@@ -0,0 +1,98 @@
+
+# EFR32 Sleepy Demo Example
+
+The EFR32 Sleepy applications demonstrates Sleepy End Device behaviour using
+the EFR32's low power EM2 mode. The steps below will take you through the
+process of building and running the demo
+
+For setting up the build environment refer to [examples/platforms/efr32/README.md](../../platforms/efr32/README.md).
+
+
+## 1. Build
+
+```bash
+$ cd <path-to-openthread>
+$ ./bootstrap
+$ make -f examples/Makefile-efr32 COMMISSIONER=1 JOINER=1 DHCP6_CLIENT=1 DHCP6_SERVER=1 BOARD=BRD4161A
+```
+
+Convert the resulting executables into S-Record format and append a s37 suffix.
+
+```bash
+$ cd output/efr32/bin
+$ arm-none-eabi-objcopy -O srec efr32-sleepy-demo-mtd efr32-sleepy-demo-mtd.s37
+$ arm-none-eabi-objcopy -O srec efr32-sleepy-demo-ftd efr32-sleepy-demo-ftd.s37
+```
+
+In Silicon Labs Simplicity Studio flash one device with the efr32-sleepy-demo-mtd.s37
+image and the other device with the efr32-sleepy-demo-ftd.s37 image.
+
+
+## 2. Starting nodes
+
+For demonstration purposes the network settings are hardcoded within the source files.
+The devices start Thread and form a network within a few seconds of powering on. In a real-life
+application the devices should implement and go through a commissioning process to create
+a network and add devices.
+
+When the efr32-sleepy-demo-ftd device is started in the CLI the user shall see:
+
+```CLI
```

Comment:
I don't think `CLI` is a supported language (see [render](https://github.com/openthread/openthread/blob/ce0f688adf14a872f463a4f3c1e025251fe069e6/examples/platforms/efr32/sleepy-demo/README.md)).